code-of-conduct.md

```diff
@@ -1,58 +0,0 @@
-## Kubernetes Community Code of Conduct
```

Comment:
brandon said he will take care of the move , see https://github.com/kubernetes/kubernetes/pull/36817